pipeline_scripts/override_tclean_commands.json

```diff
@@ -22,4 +22,36 @@
       }
     }
   }
+  "Sgr_A_st_t_03_7M": {
+    "tclean_cube_pars": {
+      "spw24": {
+        "imagename": "uid___A001_X15a0_X114.s38_0.Sgr_A_star_sci.spw24.cube.I.iter1.reclean",
+        "threshold": "0.06Jy",
+        "noisethreshold": 5.0,
+        "lownoisethreshold": 2.0,
+        "growiterations": 75
+        "sidelobethreshold": 1.25,
+        "negativethreshold": 0.0,
+        "smoothfactor": 0.5,
+        "minbeamfrac": 0.1,
+        "cutthreshold": 0.01
+      },
+    }
+  }
```

Comment:
```suggestion
  },
```

Reply:
the current version of this pull request is syntactically incorrect; a comma is needed here in order for this code to be valid JSON